.github/workflows/e2e-nextjs.yml

```diff
@@ -0,0 +1,83 @@
+name: Next.js Integration test
+
+on:
+  pull_request:
+    branches: [main]
+    # Run on release PRs only
+    paths: ['CHANGELOG.md']
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bundler: ['default', 'esbuild']
+    steps:
+      - name: Checking out zip-it-and-ship-it
+        uses: actions/checkout@v2
+        with:
+          path: zip-it-and-ship-it
+      - name: Checking out netlify-plugin-nextjs
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/netlify-plugin-nextjs
+          path: netlify-plugin-nextjs
+      - name: Checking out netlify-cli
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/cli
+          path: netlify-cli
+      - name: Installing Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Installing netlify-cli
+        run: npm ci
+        working-directory: netlify-cli
+      - name: Installing netlify-plugin-nextjs
+        run: npm install
+        working-directory: netlify-plugin-nextjs
+      - name: Installing netlify-plugin-nextjs demo
+        run: npm install
+        working-directory: netlify-plugin-nextjs/demo
+      - name: Patching netlify.toml to change the bundler
+        if: "${{ matrix.bundler != 'default' }}"
+        run:
+          'npm i @iarna/toml --no-save && node -e "const { readFileSync, writeFileSync } = require(''fs'');const {
+          parse, stringify } = require(''@iarna/toml'');const config = parse(readFileSync(''netlify.toml'',
+          ''utf-8''));config.functions = { ...config.functions, node_bundler: ''${{ matrix.bundler }}''
+          };writeFileSync(''netlify.toml'', stringify(config));"'
+        working-directory: netlify-plugin-nextjs/demo
+      - name: Symlinking zip-it-and-ship-it (1/2)
+        run:
+          rm -rf $GITHUB_WORKSPACE/netlify-cli/node_modules/@netlify/zip-it-and-ship-it && ln -s
+          $GITHUB_WORKSPACE/zip-it-and-ship-it $GITHUB_WORKSPACE/netlify-cli/node_modules/@netlify/zip-it-and-ship-it
+      - name: Symlinking zip-it-and-ship-it (2/2)
+        run:
+          rm -rf $GITHUB_WORKSPACE/netlify-cli/node_modules/@netlify/build/node_modules/@netlify/zip-it-and-ship-it &&
```

Comment:
Should we do this for `functions-utils` too?

Also should we use `npm link` instead of manually linking?

Reply:
> Should we do this for functions-utils too?

Yes, good call!

> Also should we use npm link instead of manually linking?

I did try with `npm link` but it wasn't working as expected. How would it work when linking transitive dependencies? Would we run `npm link @netlify/zip-it-and-ship-it` from within the various packages in `node_modules`?

Reply:
> Would we run `npm link @netlify/zip-it-and-ship-it` from within the various packages in `node_modules`?

Yes, I think we would need to that for each copy